pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -118,6 +171,9 @@ type BareMetalHostSpec struct {
 	// How do we connect to the BMC?
 	BMC BMCDetails `json:"bmc,omitempty"`
 
+	// RAID configuration for bare metal server
+	RAID *RAIDConfig `json:"raid,omitempty"`
```

Comment:
I would like to change `RAID` property from `RAIDConfig` to `*RAIDConfig` in order to make it be `nil` in the case of RAID is not configured from BaremetalHost yaml file.